src/lib/array-diff.js

```diff
@@ -1,19 +1,18 @@
 function difference(a, b) {
-  var hash = {}
-  var diff = {}
+  const hash = {};
+  const diff = {};
 
-  b.forEach(function createHash(item) {
-    hash[item] = true
-  })
+  b.forEach(item => {
+    hash[item] = true;
+  });
 
-  a.forEach(function findDiff(item) {
+  a.forEach(item => {
     if (!hash[item] && !diff[item]) {
-      diff[item] = true
+      diff[item] = true;
     }
-  })
-
-  return Object.keys(diff)
+  });
 
+  return Object.keys(diff);
```

Comment:
I just ran the tests, and since we're only comparing whether an item of `a` is not included in `b`, the following would suffice:

```JavaScript
function difference(a, b) {
    return a.filter(item => !b.includes(item)).sort((a, b) => a > b);
}
```